easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'MakeCp'
+
+name = 'ACTC'
+version = '1.1'
+
+homepage = 'https://sourceforge.net/projects/actc'
+description = "ACTC converts independent triangles into triangle strips or fans."
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
```

Comment:
@jdonners please include a `SHA256` checksum (you can use `eb --inject-checksums` for that)